symfony/twig-bundle/5.0/config/packages/twig.yaml

```diff
@@ -0,0 +1,4 @@
+twig:
+    default_path: '%kernel.project_dir%/templates'
```

Comment:
I think this could be removed too, as it is already the default path in 5.0 (this option being here to allow opting into the new structure in a 4.x project)

Reply:
That will apply to `debug` and `strict_variables` options.

Reply:
I also think they are not here due to of the new structure 4.0 only, but because they are also relevant options for the user who might want to change.

Reply:
We tend to not have default values when people should not change them often. I think that this is the case here. So, I would be in favor of removing it.

Reply:
So, it means that we can remove the file in 5.0?

Reply:
Technically yes, we can.

Reply:
AFAIU, `default_path` was a migration-only setting that we will deprecate again in 5.1. That's not something users should change in 5.0 (they should however change it in 4.x to opt-in for the new structure).

Reply:
@stof I don't think so, the `default_path` value was since the begining the same `%kernel.project_dir/templates%`, and it was intended to be configurable.